VictorSydorchenkoHomework1.cpp

```diff
@@ -0,0 +1,113 @@
+// Данная програмка исполняет все, указанное в задании, но я решил добавить ввод вручную, чтобы попрактиковаться.
+#include <cmath>
+#include <iostream>
+#include <stdlib.h>
```

Comment:
в С++ предпочитают стиль написания залоговочных файлов которые есть частью C писать так 
```suggestion
#include <cstdlib>
```

Reply:
Не знал, учту это и поправлю.